app/filters/AffectXFrameOptionsHeader.scala

```diff
@@ -0,0 +1,22 @@
+package filters
+
+import configuration.Config
+import controllers.Cached._
+import play.api.Play.current
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.ws.WS
+import play.api.mvc._
+
+import scala.concurrent.Future
+
+object AffectXFrameOptionsHeader extends Filter {
+  def apply(nextFilter: RequestHeader => Future[Result])(requestHeader: RequestHeader): Future[Result] = {
+    nextFilter(requestHeader).map { result =>
+      if (requestHeader.path.startsWith("/q/")) {
```

Comment:
I'd be slightly more comfortable with an X-Frame-Options-Override header or something as opposed to matching on the URL